BLaurenB/techtime
# User Story 8: Add user profile information to user show page after login

#### What's this PR do?
Adds profile information for a user directly after logging in.

#### Where should the reviewer start- what are the pertinent changes and the sequence?
1). app/views/users/show.html.erb - adds a div for ".profile" that displays user information.
2). spec/features/users/registered_user_can_log_in_spec.rb - updates test to find the profile information within the css class of profile

#### How should this be manually tested?
Start the server and log in; the landing page should have information about the account that just logged in.

#### Any background context you want to provide?
I'm not sure why 9 commits are on this PR; 2 of them brought in changes from dev, and then 1 commit actually made these changes.

The other 6 commits are from a different PR.... :?: I still have work to do on my git workflow 😸 

#### What are the relevant tickets?
Closes #100 
#### Screenshots (if appropriate)
#### Questions:
